nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -190,6 +210,19 @@ def forward(self, x, pad_mask=None):
         x = x.transpose(1, 2)
         return x
 
+    def reset_parameters_conv(self):
+        pw1_max = self.d_model ** -0.5
```

Comment:
What are there two pw1_max and pw2_max?

Reply:
Just named for the two pointwise ops, can be merged into one line. Done.